src/ElementUtilities.js

```diff
@@ -21,7 +21,7 @@ const findClosest = (el, selector) => {
  * @return {boolean}        Whether table ancestor of 'el' is found
```

Comment:
Do the same thing for other methods elsewhere including WidenImage.widenAncestors().